src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -0,0 +1,218 @@
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.DHTxx
+{
+    public class DelayMS
```

Comment:
I feel like this will need to be either internal or renamed (we usually don't use abbreviations)